src/data/README.md

```diff
@@ -18,7 +18,235 @@ TODO - describe the properties used in DevDot
+
+```json5
+{
+  "heading": "string", // required
+  "subheading": "string", // required
+  "blocks": [ /*  array of objects, required */ ], 
+}
+```
+
+Each of these top-level properties is addressed in more detail below.
+
+#### `heading` and `subheading`
+
+Set the heading and subheading shown on the page.
+
+<details>
+<summary>Example</summary>
+
+Source: 
+
+
+```json5
+{
+  "heading": "Waypoint Documentation", 
+  "subheading": "Use Waypoint to deliver a PaaS-like experience for Kubernetes, ECS, and other platforms.", 
+  "blocks": [ /* ... */ ],
+}
+```
+
+Result:
+
+![](https://user-images.githubusercontent.com/4624598/158818382-e78ea677-85c1-41aa-92b4-ca8714f06f2d.png)
+
+</details>
+
+#### `blocks`
+
+Each item in the `blocks` array represents a component on the page. Each of these items must have a `type` property, which can be one of the types listed below.
+
+<!-- block: type heading -->
+
+<details>
+<summary>Block type: <code>"heading"</code></summary>
+
+Heading blocks render HTML heading elements. Each block accepts the following properties:
+
+| Property | Type | Details |
+| --- | --- | --- |
+| `type` | `"heading"` | Block type |
+| `heading` | `string` | Text for the heading |
+| `level` | `2 \| 3 \| 4 \| 5 \| 6` | Semantic heading level, for example `2` becomes `<h2>`. Note that there is already an `<h1>` rendered for the page, so only values  should be used. |
+| `size` | `100 \| 200 \| 300 \| 400 \| 500` | Visual size of the heading. `500` is the largest size and `100` is the smallest. Visual size should generally reflect the semantic level, with `h2 = 300`, `h3 = 200`, and `h4` and below at the `100` size.
+
+Example: `h2` with `300` size:
+
+```json5
+{
+  "type": "heading",
+  "heading": "Featured Reference Docs",
+  "level": 2,
+  "size": 300,
+}
+```
+
+![](https://user-images.githubusercontent.com/4624598/158818745-a20d1892-efa6-4053-9d00-811645d642aa.png)
+
+Example: `h2` with `400` size:
+
+```json5
+{
+  "type": "heading",
+  "heading": "Explore Waypoint Documentation",
+  "level": 2,
+  "size": 400,
+}
+```
+
+![](https://user-images.githubusercontent.com/4624598/158818617-2b8ce029-ad41-4081-8701-869d51abf40b.png)
+
+</details>
+
+<!-- block type: get_started -->
+
+<details>
+<summary>Block type: <code>"get_started"</code></summary>
+
+The `"get_started"` block renders a heading, descriptive text, and a single link alongside a product icon.
+
+| Property | Type | Details |
+| --- | --- | --- |
+| `type` | `"get_started"` | Block type |
+| `product` | `"boundary" \| "consul" \| "nomad" \| "packer" \| "terraform" \| "vault" \| "vagrant" \| "waypoint" \| "sentinel" \| "hcp"` | Product icon to be shown. |
+| `heading` | `string` | Text for the heading |
+| `text` | `string` | Descriptive text shown below the heading |
+| `link` | `{ text: string, url: string }` | Link shown below the body text |
```

Comment:
💭 Do you think referencing the `StandaloneLink` here is too much info?

```suggestion
| `link` | `{ text: string, url: string }` | `StandaloneLink` shown below the body text |
```

Reply:
I think it's a good suggestion! Still clear to authors that it's a link, clearer to us that it's a specific component being used 👍